tests/unit/test_compose_manager.py

```
@@ -0,0 +1,420 @@
+
+            assert result["host_id"] == "test-host-1"
+            assert result["stacks_found"] == []
+            assert "error" in result["analysis"].lower()
+
+
+@pytest.mark.unit
+@pytest.mark.asyncio
+class TestWriteComposeFile:
+    """Tests for write_compose_file method."""
+
+    async def test_write_compose_file_success(self, docker_mcp_config, mock_docker_context_manager, tmp_path):
```

Ruff (0.14.4)

Unused method argument: `tmp_path`

(ARG002)